packages/@aws-cdk/cfnspec/build-tools/create-missing-libraries.ts

```diff
@@ -102,12 +102,12 @@ async function main() {
     console.log(`generating module for ${packageName}...`);
 
     await write('package.json', {
-      name: packageName,
+      'name': packageName,
```

Comment:
Please, no! I have a *strong* preference for omitting the quotes here unless they're absolutely necessary.

Reply:
This is required by [quote-prop `consistent-as-needed`](https://eslint.org/docs/rules/quote-props), the ` 'cdk-build'` property needs to be quoted.
I'm not sure why this wasn't previously picked up by TSLint.

Reply:
Ohhhh I missed it was there!!! Then Yes, definitely YES!